code/game/objects/items/storage/pouch.dm

```diff
@@ -58,29 +58,13 @@
 	max_w_class = SIZE_MEDIUM
 	cant_hold = list( //Prevent inventory bloat
 		/obj/item/storage/firstaid,
-		/obj/item/storage/bible
+		/obj/item/storage/bible,
+		/obj/item/storage/fancy,
```

Comment:
This restricts you from putting stuff like cigs, crayons, and donuts in the pouch, if you want to stop vial boxes you can use `/obj/item/storage/fancy/vials`.